pylint/lint/pylinter.py

```diff
@@ -1276,21 +1275,29 @@ def _report_evaluation(self):
         # syntax error preventing pylint from further processing)
         note = None
         previous_stats = config.load_results(self.file_state.base_name)
-        if self.stats["statement"] == 0:
+        if self.stats.statement == 0:
             return note
 
         # get a global note for the code
         evaluation = self.config.evaluation
         try:
-            note = eval(evaluation, {}, self.stats)  # pylint: disable=eval-used
+            stats_dict = {
+                "error": self.stats.error,
+                "warning": self.stats.warning,
+                "refactor": self.stats.refactor,
+                "convention": self.stats.convention,
+                "statement": self.stats.statement,
+            }
+            note = eval(evaluation, {}, stats_dict)  # pylint: disable=eval-used
```

Comment:
Not linked directly to this MR but there must be a way to not use eval here.

Reply:
Want me to add a fixme? I think it will be quite difficult though, since the setting in `pylintrc` allows users to create their own "score-function". So we will need to evaluate that function somehow.

Reply:
You're right I thought it was for our own function.